test/PostageStamp.test.ts

```diff
@@ -52,6 +53,8 @@ describe('PostageStamp', function () {
   describe('when deploying contract', function () {
     beforeEach(async function () {
       await deployments.fixture();
+      const priceOracleRole = await read('PostageStamp', 'PRICE_ORACLE_ROLE');
```

Comment:
why are these grant roles needed here? is it not handled in `004_deploy_roles_postage.ts`?

Reply:
Yes, those 2 lines for that test are actually not needed, and have just removed them.

Reply:
Also for the whole test take into account that some roles are set to contract addresses, so when you test something you got simulate as it was called from the contract.